third_party/xla_client/xrt_computation_client.h

```diff
@@ -114,22 +114,32 @@ class XrtComputationClient : public ComputationClient {
       return task_no == rhs.task_no && name == rhs.name;
     }
 
+    std::string ToString() const {
+      return name + ":" + std::to_string(task_no);
+    }
+
+    friend std::ostream& operator<<(std::ostream& stream,
+                                    const Worker& worker) {
+      stream << worker.name << ":" << worker.task_no;
```

Comment:
nit, 
```
stream << worker.ToString();
```

This way we only need to update one place when we want to change the worker string format.

Reply:
Thanks, done.